heat-stack/app/routes/_heat+/index.tsx

```diff
@@ -4,7 +4,7 @@ export default function HEATLandingPage() {
 	return (
 		<div className="flex flex-col min-h-screen">
 			{/* Hero Section */}
-			<section className="bg-gradient-to-r from-emerald-600 to-teal-500 py-20 px-4 text-white">
+			<section className="bg-gradient-to-r from-emerald-600 to-teal-500 py-20 px-4 text-white" style={{ borderRadius: '10px' }}>
```

Comment:
We might consider using `className="rounded-[10px]"` here but it is between the lg and xl rounded corner presets.